core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt

```diff
@@ -46,7 +40,7 @@ class FakerConfig private constructor(
      * Default: `100`.
      */
     @FakerDsl
-    class Builder internal constructor() {
```

Comment:
Any reason to make the constructor public?

Reply:
Yes. It's not possible to make the class `internal` or `private` when it's used as a parameter.

![image](https://user-images.githubusercontent.com/13183702/118513800-578d6a00-b734-11eb-8602-d50cfaaf845f.png)

Reply:
Good point. I made them internal again. I guess I overlooked them.